packages/system/src/stories.mdx

```diff
@@ -0,0 +1,49 @@
+import { Meta, Story, Preview } from '@storybook/addon-docs/blocks';
+
+<Meta title="MDX/How to Write Components" />
```

Comment:
Should we put this in a "guides" folder?

Reply:
Makes sense if we have more guides at some point :+1: